Library/Homebrew/cmd/update-bash.sh

```diff
@@ -208,6 +208,12 @@ update-bash() {
     esac
   done
 
+  if ! curl -I --connect-timeout 2 https://github.com &> /dev/null
```

Comment:
`brew.sh` is pretty fast (0.2-0.4 seconds) but I wasn't sure on how much we'd :heart: pinging a non-SSL/TLS address every time `update` is ran (when it's rolled out beyond us lot). Potentially it'd allow people to sniff out likely Homebrew users on public wifi.

No objection to the higher timeout. I put it low to avoid people thinking it was unintentionally hanging and reporting it an an Issue.